COMPILING.md

```diff
@@ -482,3 +482,42 @@ successfully on Windows or macOS.
    The argument for the IPASIR parameter gives the build system the location for
    the IPASIR headers, which is needed for the cbmc includes of `ipasir.h`. The
    compiled binary will be placed in `cbmc/src/cbmc/cbmc`.
+This document assumes you have already been able to build CPROVER on
+your chosen architecture. For details on building CPROVER see
+COMPILING.md.
```

Comment:
Since this content is _in_ `COMPILING.md` - maybe its better to say `see section BLAH` ?

Reply:
Fixed